en/installconfig-add-nodes.md

```diff
@@ -40,67 +39,53 @@ A node configured to boot via PXE will:
 
 During this process, the MAAS server will be passed information about the node,
 including the architecture, MAC address and other details which will be stored
-in the database.
+in the database. This information-gathering process is known as *automatic
+discovery*.
 
 
 ## KVM guest nodes
 
-If your MAAS nodes will be backed by KVM guests, ensure the `virsh` binary is
-available to the rack controller via the `libvirt-bin` package. In the UI,
-the power type will be 'virsh'.
-
-!!! Note: The minimum amount of memory per guest should be 768 MB.
-
-The virsh power type takes two parameters:
-
-`Power ID`: This is the name of the virtual machine (libvirt "domain") shown
-by:
-
-```bash
-sudo virsh list --all
-```
-
-`Address:` This is a libvirt connection string, such as:
-
-```nohighlight
-qemu+ssh://ubuntu@10.0.0.2/system
-```
-
-![qemu ssh power](../media/virsh-config.png)
+KVM-backed nodes are common and so a little extra guidance is provided here.
+Begin by ensuring the `virsh` binary is available to the rack controller via
+the `libvirt-bin` package.
 
-For SSH, you'll need to generate an SSH keypair for the 'maas' user. A home
-directory and a login shell will also need to be set up:
+The 'maas' user will need an SSH keypair (with a null passphrase) so MAAS will
+be able to query and manage KVM guests remotely. A login shell will also be
+useful when becoming user 'maas':
 
 ```bash
-sudo mkdir /home/maas
-sudo chown maas:maas /home/maas
 sudo chsh -s /bin/bash maas
+sudo su - maas
+ssh-keygen -f ~/.ssh/id_rsa -N ''
 ```
 
-Become the 'maas' user and generate a keypair with the private key having a
-null passphrase:
+Add the public key to `/ubuntu/.ssh/authorized_keys` on the KVM host:
```

Comment:
I know this is inherited from the old docs, but I guess this should be /home/<USER>/.ssh/authorized_keys on the KVM host, where <USER> is the user managing the KVM guests.